src/components/SecondaryPanes/Expressions.css

```diff
@@ -6,12 +6,20 @@
   cursor: pointer;
   color: var(--theme-body-color);
   background-color: var(--theme-body-background);
-  line-height: 12px;
+  font-size: 12px;
 }
 
 .input-expression::-webkit-input-placeholder {
   text-align: center;
   font-style: italic;
+  color: var(--theme-body-color);
+}
+
+.input-expression::-moz-placeholder {
```

Comment:
maybe we could group those 2 rules together ? 

```css
.input-expression::-webkit-input-placeholder,
.input-expression::-moz-input-placeholder,
```
I even think we can drop the `-moz` prefix since it has been unprefixed in Firefox 51 (https://developer.mozilla.org/en-US/docs/Web/CSS/::placeholder)